CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.3.1] - 2021-01-28
+
+### Fixed
+
+- Fix potential integer overflow detected by oss-fuzz
+- Fix #615 using libraqm
+- Fix #303: gdlib.pc: use Requires instead of Libs (#537)
+- webp: support pkg-config file
+- gd_io: replace internal Putchar with gdPutC
+- gd_io: trim unused Putword function
+- Fix indent error: travis/main.sh
+- Fixed #472: Adjusting CMakeLists.txt (#582)
+- Fix #615: gdImageStringFT() fails for empty strings as of libgd 2.3.0 (#633)
+- Fix typo but preserve BC
+- Fixed test case build configure error in issue #626
+- Fix two build configure errors in tests (#616)
+- .travis.yml: close config warning and info (#642)
```

Comment:
the changelog is meant as release notes readable by the end user to know what has changed.  we should omit entries like this that only impact the CI and have no bearing on the release package.